bigframes/_config/bigquery_options.py

```diff
@@ -33,19 +33,22 @@ class BigQueryOptions:
 
     def __init__(
         self,
+        *,
```

Comment:
Any chance we can separate the breaking change with others? Then the commit history will be more user-friendly.

Reply:
Reverted the signature change.